CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ cmake_minimum_required(VERSION 3.2.2)
 project(leatherman VERSION 0.12.0)
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+# Populate locale install location
+configure_file(cmake/leatherman.cmake.in "${CMAKE_CURRENT_SOURCE_DIR}/cmake/leatherman.cmake" @ONLY)
```

Comment:
I'd rather this were written into CMAKE_BINARY_DIR (and we added that to the CMAKE_MODULE_PATH and installed from there as well).

We haven't been good about that for generated files, but it makes it difficult to have more than one build of the same project. I'd like to move towards cleaner isolation of generated files.